weldx/measurement.py

```diff
@@ -1,9 +1,11 @@
 """Contains measurement related classes and functions."""
 
 from dataclasses import dataclass, field
-from typing import List, Union  # noqa: F401
+from typing import Dict, List, Tuple, Union  # noqa: F401
 
+import matplotlib.pyplot as plt
```

Comment:
please import matplotlib and networkx lazily (inside the method bodies) as these pkgs dramatically increase the import time of weldx.